src/coreclr/src/jit/compiler.h

```diff
@@ -66,6 +66,9 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 // a verification block should be inserted
 #define SEH_VERIFICATION_EXCEPTION 0xe0564552 // VER
 
+// Invalid offset marker value for dynamic dictionary expansions support
+#define EXPRUNTIMELOOKUP_INVALID_OFFSET 0xFFFF
```

Comment:
Why is this not using `CORINFO_SKIPSIZECHECK` ?

Reply:
All the changes related to this feature on the JIT side had the "expruntimelookup" naming, so I wanted to follow the same convention on the JIT side for consistency. Would it be better to change it to CORINFO_NO_SIZE_CHECK?

Reply:
Yes. The JIT should be using the constants from corinfo.h. It should not be duplicating them with different names.